### ISSUE TITLE
Spectral PMN with maximum decibel values per frequency bin

### ISSUE THREAD

**Issue by ThomasLuypaertGitHub** [QutEcoacoustics/audio-analysis#293]
## Is your feature request related to a problem? Please describe.
At present, the spectral PMN index provides the averaged decibel value per frequency bin. 
## Describe the solution you'd like
It would be nice to have an option to calculate the maximum decibel value per frequency bin as well.

**Comment by atruskie** [comment]
To be clear:

You want the *maximum* decibel value from each frequency bin, after noise removal?

What should we name this? MMN?

Currently, PMN is the *average* decibel value from each frequency bin, after noise removal.

**Comment by ThomasLuypaertGitHub** [comment]
> To be clear:
> 
> You want the _maximum_ decibel value from each frequency bin, after noise removal?
> 
> What should we name this? MMN?
> 
> Currently, PMN is the _average_ decibel value from each frequency bin, after noise removal.

Yes, indeed, the _maximum_ decibel value for each frequency bin after noise removal. 

If the Signal-to-Noise ratio is currently defined as "_The difference between the maximum decibel value in the decibel envelope and the decibel value of BGN_", perhaps this new feature could be the spectral SNR, or SSN/SNRsp?

**Comment by towsey** [comment]
Yes, I think spectral SNR is a reasonable name for this index. I can set this up when I work on Issue 292, calculating the spectral centroid as a summary index.
